services/openvpn/client_config.go

```diff
@@ -100,9 +100,11 @@ func NewClientConfigFromSession(sessionConfig []byte, configDir string, runtimeD
 	if vpnConfig.LocalPort > 0 {
 		clientFileConfig.OriginalRemoteIP = vpnConfig.RemoteIP
 		clientFileConfig.OriginalRemotePort = vpnConfig.RemotePort
-		vpnConfig.RemoteIP = "127.0.0.1"
-		// TODO: randomize this too?
-		vpnConfig.RemotePort = vpnConfig.LocalPort + 1
+		if isMobile {
```

Comment:
Could we document the purpose of this change in the git message body?

Reply:
Moved mobile `vpnConfig` factoring to same place